testplan/exporters/testing/pdf/__init__.py

```diff
@@ -144,6 +145,11 @@ def create_pdf(source, config):
             reportlab_data.extend(row_data.content)
             reportlab_styles.extend(row_data.style)
 
+    pdf_path = config.pdf_path
+    if os.path.exists(pdf_path):
```

Comment:
How to determinate the current runnable in the exporter class?

Reply:
> This might be valid in the case when you save from interactive, as you get a link/button for that. When one just run it from commandline I prefer the override behaviour, than generationg something I need to search for.

How to determinate the current runnable in the exporter class?

Reply:
not necessary in the exporter class, might be in the export function which could take a parameter describing the nature of export.